src/utilities/util_dump.c

```diff
@@ -769,28 +769,32 @@ dump_record(
     int (*bck)(WT_CURSOR *);
     int (*fwd)(WT_CURSOR *);
     int exact;
-    const char *current_key;
     bool once;
-
     session = cursor->session;
+    WT_DECL_ITEM(tmp);
+    WT_RET(__wt_scr_alloc((WT_SESSION_IMPL *)session, 0, &tmp));
```

Comment:
New line between declaration and mallocing.

Reply:
Changed in most recent commit